src/state_point.F90

```diff
@@ -330,11 +330,11 @@ subroutine write_state_point()
           MOMENT_LOOP: do j = 1, tally % n_user_score_bins
             select case(tally % score_bins(k))
             case (SCORE_SCATTER_N, SCORE_NU_SCATTER_N)
-              str_array(k) = 'P' // trim(to_str(tally % moment_order(k)))
+              str_array(k) = trim(to_str(tally % moment_order(k)))
```

Comment:
My issue with it is related to my rationale behind using explicit scores "scatter-0", ... "scatter-N" above. If you specify a "scatter-1" tally the output in the statepoint is "scatter-P1". It seems to me that it would be better for the output to match the input so that one can easily find a `Tally` created for the input in the output (which is not the case with these "P"'s added to the scores).